<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelStringCompleter</class>
 <widget class="QFrame" name="CAircraftModelStringCompleter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="hl_ModelStringCompleter">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_modelString">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>model string</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_SourceSelection" native="true">
     <layout class="QHBoxLayout" name="hl_SourceSelection">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_Db">
        <property name="text">
         <string>DB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_ModelSet">
        <property name="toolTip">
         <string>own model set</string>
        </property>
        <property name="text">
         <string>set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_OwnModels">
        <property name="toolTip">
         <string>all installed models</string>
        </property>
        <property name="text">
         <string>inst.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
